src/components/KMethod/KMethod.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <KBadge
+    :background-color="backgroundColor || methodBadgeColors[method].backgroundColor"
+    class="k-method"
+    :class="`method-${method}`"
+    :color="color || methodBadgeColors[method].color"
+    max-width="auto"
+    :shape="appearance === 'rectangular' ? 'rectangular' : 'rounded'"
+  >
+    {{ methodLabel }}
+  </KBadge>
+</template>
+
+<script lang="ts">
+import { PropType, computed } from 'vue'
+import type { Method, MethodAppearance } from '@/types'
+
+export const methods: Method[] = ['get', 'post', 'put', 'patch', 'delete', 'head', 'options', 'trace', 'connect', 'custom']
+
+export const shapes: MethodAppearance[] = ['rounded', 'rectangular']
+</script>
+
+<script setup lang="ts">
+const props = defineProps({
+  method: {
+    type: String as PropType<Method>,
+    required: true,
+    validator: (value: string): boolean => {
+      return methods.includes(value as Method)
+    },
+    default: 'post',
+  },
+  appearance: {
+    type: String as PropType<MethodAppearance>,
+    required: false,
+    validator: (value: string): boolean => {
+      return shapes.includes(value as MethodAppearance)
+    },
+    default: 'rounded',
+  },
+  label: {
+    type: String,
+    required: false,
+    default: '',
+  },
+  color: {
+    type: String,
+    required: false,
+    default: '',
+  },
+  backgroundColor: {
+    type: String,
+    required: false,
+    default: '',
+  },
```

Comment:
question: if the purpose of this component is to standardize the method badge colors, why would we provide props to allow overriding the colors? 

If these props are being used, they may as well just use the native KBadge component. 

Same question for the `appearance` prop

Reply:
> Well so even though we are building this component for our own use case, open source users might want to be able to configure something custom

Then wouldn't they just use `KBadge`? Literally the only difference in this component is the colors, right?

Reply:
I disagree with this, we've had this discussion before. Whether or not it's pure, Konnect frequently drives our Kongponents. We have three use cases in Konnect for the method variants and they all follow this color scheme. We need a place where all of these components can be included and it doesn't make sense to have two places where we have to maintain these color mappings. If you don't think kongponents is the right repo, fair enough, but we need a component like this in some repo somewhere.

Reply:
I'll defer on the toggle; however, I still believe the color props should be removed.